src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -14,8 +14,11 @@
 
   private int torpedoCount = 0;
 
+  private Random generator;
+
   public TorpedoStore(int numberOfTorpedos){
     this.torpedoCount = numberOfTorpedos;
+    generator = new Random();
```

Comment:
This is the proper way of using the Random class to generate random numbers! Good job!